Pipfile

```diff
@@ -14,7 +14,6 @@ botocore = "==1.12.162"
 click = "==6.7"
 click-repl = "==0.1.6"
 docutils = "==0.14"
-futures = "==3.2.0"
```

Comment:
This is a backport of https://docs.python.org/3/library/concurrent.futures.html so unneeded on Python 3 AFAICT